pkg/adapterManager/manager.go

```diff
@@ -235,11 +273,37 @@ func (m *Manager) runAsync(ctx context.Context, requestBag, responseBag *attribu
 	cfg *cpb.Combined, invokeFunc invokeExecutorFunc, resultChan chan result) {
 	df, _ := m.df.Load().(descriptor.Finder)
 	m.gp.ScheduleWork(func() {
+
+		// tracing
+		op := fmt.Sprintf("%s:%s(%s)", cfg.Aspect.Kind, cfg.Aspect.Adapter, cfg.Builder.Impl)
+		span, ctx := opentracing.StartSpanFromContext(ctx, op)
```

Comment:
Oh, btw, this is another reason to use the (to be finalized and merged) `tracing.Report` facade: we can do things like avoid evaluating string format args when we know that tracing is disabled.